packages/integrations/src/ember.ts

```diff
@@ -57,36 +56,17 @@ export class Ember implements Integration {
       'error',
       (reason: any): void => {
         if (getCurrentHub().getIntegration(Ember)) {
-          getCurrentHub().withScope(scope => {
+          getCurrentHub().withScope(() => {
```

Comment:
```suggestion
          getCurrentHub().withScope(scope => {
```

this one is actually still needed